docs/marketplace/user/features/202108.0/marketplace-merchant-feature-overview/marketplace-merchant-feature-overview.md

```diff
@@ -133,8 +111,8 @@ If the merchant is not active, their products and offers are not displayed in th
 
 |FEATURE OVERVIEWS  |MERCHANT PORTAL USER GUIDES  |BACK OFFICE USER GUIDES |
 |---------|---------|---------|
-|[Merchant users overview](/docs/marketplace/user/features/{{ page.version }}/marketplace-merchant-feature-overview/merchant-users-overview.html) | [Editing merchant’s profile details](/docs/marketplace/user/merchant-portal-user-guides/{{ page.version }}/profile/editing-merchants-profile-details.html) |[Managing merchants](/docs/marketplace/user/back-office-user-guides/{{ page.version }}/marketplace/merchants/managing-merchants.html)|
-|| | [Managing merchant users](/docs/marketplace/user/back-office-user-guides/{{ page.version }}/marketplace/merchants/managing-merchant-users.html)|
+|[Merchant users overview](/docs/marketplace/user/features/{{ page.version }}/marketplace-merchant-feature-overview/merchant-users-overview.html) | [Editing merchant's profile details](/docs/marketplace/user/merchant-portal-user-guides/{{ page.version }}/profile/editing-merchants-profile-details.html) |[Managing merchants](/docs/marketplace/user/back-office-user-guides/{{ page.version }}/marketplace/merchants/managing-merchants.html)|
+|[Marketplace inventory management feature overview](/docs/marketplace/user/features/{{ page.version }}/marketplace-inventory-management-feature-overview.md)| | [Managing merchant users](/docs/marketplace/user/back-office-user-guides/{{ page.version }}/marketplace/merchants/managing-merchant-users.html)|
```

Comment:
Delete link to the inventory management feature overview here. In this table, we list only overviews that belong to this specific feature (marketplace merchants in this case). If you want to add link to this guide, this should be done as the "related article" link